plugins/source/gcp/policies_v1/cis_v1.2.0/section_3.sql

```diff
@@ -10,7 +10,7 @@
 \ir ../queries/compute/legacy_network_exist.sql
 \set check_id '3.3'
 \echo "Executing check 3.3"
-\ir ../queries/compute/dnssec_disabled.sql
```

Comment:
What are we planning on doing with the legacy query - are we planning on removing the file or leaving it?  Is it linked anywhere else?

Reply:
I think I'll remove this query, since it doesn't appear to be useful at all.